lib/msal-browser/src/telemetry/PerformanceManager.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Logger } from "@azure/msal-common";
+import { PerformanceMeasurement } from "./PerformanceMeasurement";
+
+export class PerformanceManager {
```

Comment:
This should probably end up being `BrowserPerformanceManager`, with an abstract `PerformanceManager` in common.

Reply:
And come up with a name other than "manager"